physics/satmedmfvdifq.F

```diff
@@ -242,6 +249,8 @@ subroutine satmedmfvdifq_run(im,km,progsigma,ntrac,ntcw,ntrw,     &
       real(kind=kind_phys) qlcr, zstblmax, hcrinv
 !
       real(kind=kind_phys) h1
+
+      real(kind=kind_phys) bfac, mffac
```

Comment:
It looks like bfac is only ever set to a constant value. Why not make it a parameter. Also, is there any documentation in the code about what bfac and mffac represent or why their values are set the way they are? Is it really necessary to set bfac to 100 in three different places?

Reply:
@chxmoonlight or @AndrewHazelton, could you help to answer/address @grantfirl's comments/questions here? Thanks!

Reply:
Xiaomin posted the reference about the parameter below. I think you are right that it should only be set once, however. We can modify that.